src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.component.html

```diff
@@ -88,6 +88,19 @@ <h4 *ngIf="!modelingExercise.id; else editExercise" id="myModelingExerciseLabel"
                 <option value="DeploymentDiagram">{{ 'artemisApp.DiagramType.DeploymentDiagram' | translate }}</option>
             </select>
         </div>
+        <div class="form-group" *ngIf="this.modelingExercise.diagramType === DiagramType.ClassDiagram || this.modelingExercise.diagramType === DiagramType.ActivityDiagram">
+            <div class="form-check custom-control custom-checkbox">
+                <input
+                    type="checkbox"
+                    id="automatic_assessment_enabled"
+                    [ngModel]="modelingExercise.assessmentType === AssessmentType.SEMI_AUTOMATIC"
+                    (ngModelChange)="modelingExercise.assessmentType = $event ? AssessmentType.SEMI_AUTOMATIC : AssessmentType.MANUAL"
+                    class="form-check-input custom-control-input"
+                    name="automaticAssessmentEnabled"
+                />
+                <label class="form-check-label custom-control-label" for="automatic_assessment_enabled" jhiTranslate="artemisApp.textExercise.automaticAssessmentEnabled"></label>
```

Comment:
Should we refactor the string away from text?

Reply:
I think this is fine for now